docs/how-to-guides/run-chainhook-as-a-service-using-stacks.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Run Chainhook as a Service using Stacks
+---
+
+# Run Chainhook as a Service using Stacks
+
+This document helps you configure the Stacks and chainhook nodes to extract on-chain data based on the predicate definition.
+
+Start with the prerequisite section and configure your files to start the chainhook service.
+
+## Prerequisite
+
+### Configure Stacks Node
+
+- Configure your stacks node using the [Stacks node configuration](https://docs.stacks.co/docs/nodes-and-miners/stacks-node-configuration) documentation.
+- Recommend the latest version of Stacks. You can check the latest version by following [this](https://github.com/stacks-network/stacks-blockchain/releases) link.
+- Get the rpcuser, rpcpassword, and rpc_port defined in the `bitcoin.conf` file from [this section](run-chainhook-as-a-service-using-bitcoind.md#prepare-the-bitcoind-node) of the [running chainhook as a service using bitcoind](run-chainhook-as-a-service-using-bitcoind.md) to use in this article.
+
+A `stacks.toml` file gets generated when you configure the stacks node, as shown below. Ensure that the `username`, `password`, and `rpc_port` values match the values in the `bitcoin.conf` file. Also, note the `rpc_bind` port to use in the `chainhook.toml` configuration in the next section of this article.
```

Comment:
The file that's generated is `Chainhook.toml`, could you make that change throughout this file?

Reply:
Done! Thanks!